.github/workflows/deploy.yml

```
@@ -0,0 +1,101 @@
+name: Deploy
+on:
+  push:
+    branches: [ "release" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)